frontend/config/webpack.common.js

```diff
@@ -35,71 +36,78 @@ module.exports = env => {
               loader: 'ts-loader',
               options: {
                 transpileOnly: env !== 'development',
-                experimentalWatchApi: true
-              }
-            }
-          ]
+                experimentalWatchApi: true,
+              },
+            },
+          ],
         },
         {
           test: /\.(svg|ttf|eot|woff|woff2)$/,
           // only process modules with this loader
           // if they live under a 'fonts' or 'pficon' directory
           include: [
             path.resolve(RELATIVE_DIRNAME, 'node_modules/patternfly/dist/fonts'),
-            path.resolve(RELATIVE_DIRNAME, 'node_modules/@patternfly/react-core/dist/styles/assets/fonts'),
-            path.resolve(RELATIVE_DIRNAME, 'node_modules/@patternfly/react-core/dist/styles/assets/pficon'),
+            path.resolve(
+              RELATIVE_DIRNAME,
+              'node_modules/@patternfly/react-core/dist/styles/assets/fonts',
+            ),
+            path.resolve(
+              RELATIVE_DIRNAME,
+              'node_modules/@patternfly/react-core/dist/styles/assets/pficon',
+            ),
             path.resolve(RELATIVE_DIRNAME, 'node_modules/@patternfly/patternfly/assets/fonts'),
-            path.resolve(RELATIVE_DIRNAME, 'node_modules/@patternfly/patternfly/assets/pficon')
+            path.resolve(RELATIVE_DIRNAME, 'node_modules/@patternfly/patternfly/assets/pficon'),
+            path.resolve(RELATIVE_DIRNAME, 'node_modules/monaco-editor'),
```

Comment:
This & the plugin being added are the only change on this file -- everything else is formatting -- my IDE formats to sane values without the linter's help.